test/docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:18.04
+
+ARG IAF_VERSION=7.6-20200915.172517
+ARG ORACLE_DB_IP
+
+WORKDIR /home
+RUN apt-get update
+RUN apt-get install --yes wget
+RUN apt-get install --yes zip
+
+# Modify the database URL within the Frank!Framework
+RUN mkdir work
+RUN mkdir work/iaf
+RUN wget -O work/iaf/iaf-test.war "https://nexus.ibissource.org/content/groups/public/org/ibissource/ibis-adapterframework-test/${IAF_VERSION}/ibis-adapterframework-test-${IAF_VERSION}.war"
+RUN unzip -d work/iaf work/iaf/iaf-test.war
+RUN rm work/iaf/iaf-test.war
+RUN sed -i -e "s|localhost|${ORACLE_DB_IP}|g" work/iaf/META-INF/context.xml
```

Comment:
Heb je ook geprobeerd of het ook werkt als je de localhost laat staan?
Als je database container ook op dezelfde machine draait als de ibis-container, dan zou dat toch moeten kunnen?